sunpy/instr/goes.py

```diff
@@ -242,26 +342,30 @@ def goes_chianti_tem(longflux, shortflux, satellite=8,
 
     References
     ----------
-    .. [1] White, S. M., Thomas, R. J., & Schwartz, R. A. 2005, Sol. Phys.,
-       227, 231
-    .. [2] Mazzotta, P., Mazzitelli, G., Colafrancesco, S., & Vittorio, N.
-       1998, A&AS, 133, 339
+    .. [1] White, S. M., Thomas, R. J., & Schwartz, R. A. 2005,
+        Sol. Phys., 227, 231
+    .. [2] Mazzotta, P., Mazzitelli, G., Colafrancesco, S., &
+        Vittorio, N. 1998, A&AS, 133, 339
 
     Examples
     --------
-    >>> longflux = np.array([7e-6, 7e-6])
-    >>> shortflux = np.array([7e-7, 7e-7])
-    >>> temp, em = goes_chianti_tem(longflux, shortflux, satellite=15,
-                                    date='2014-04-16', abundances="coronal")
+    >>> from astropy.units.quantity import Quantity
+    >>> longflux = Quantity([7e-6, 7e-6], unit="MK")
+    >>> shortflux = Quantity([7e-7, 7e-7], unit="cm**(-3)")
```

Comment:
using `Quantity` has less overhead than multiplying by a unit and has exactly the same result.

However, seeing as this is an example, I would agree that multiplying by a unit is the user facing idiom.